### ISSUE TITLE
1.11.2 - Crash on fresh instance install, no other mods present

### ISSUE THREAD

**Issue by DarthJimBob** [LatvianModder/FTBUtilities#454]
`---- Minecraft Crash Report ----
// Oops.

Time: 5/18/17 12:46 PM
Description: There was a severe problem during mod loading that has caused the game to fail

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from FTBUtilities (ftbu)
Caused by: java.lang.NoClassDefFoundError: mcjty/lib/tools/ItemStackTools
	at com.feed_the_beast.ftbl.lib.client.ImageProvider.get(ImageProvider.java:89)
	at com.feed_the_beast.ftbl.lib.client.ImageProvider.get(ImageProvider.java:80)
	at com.feed_the_beast.ftbl.lib.client.ImageProvider.get(ImageProvider.java:75)
	at com.feed_the_beast.ftbu.gui.Guides.loadTree(Guides.java:264)
	at com.feed_the_beast.ftbu.gui.Guides.func_110549_a(Guides.java:146)
	at net.minecraft.client.resources.SimpleReloadableResourceManager.func_110542_a(SimpleReloadableResourceManager.java:122)
	at com.feed_the_beast.ftbu.client.FTBUClient.postInit(FTBUClient.java:45)
	at com.feed_the_beast.ftbu.FTBU.onPostInit(FTBU.java:61)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:641)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:246)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:224)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:147)
	at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:814)
	at net.minecraftforge.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:343)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:521)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:352)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:236)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:297)
	at org.multimc.EntryPoint.listen(EntryPoint.java:162)
	at org.multimc.EntryPoint.main(EntryPoint.java:53)
Caused by: java.lang.ClassNotFoundException: mcjty.lib.tools.ItemStackTools
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 53 more
Caused by: java.lang.NullPointerException
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182)
	... 55 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.11.2
	Operating System: Linux (amd64) version 4.4.0-66-generic
	Java Version: 1.8.0_121, Oracle Corporation
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 558018200 bytes (532 MB) / 1167589376 bytes (1113 MB) up to 4772593664 bytes (4551 MB)
	JVM Flags: 2 total; -Xms512m -Xmx5120m
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.38 Powered by Forge 13.20.0.2228 6 mods loaded, 6 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJ	minecraft{1.11.2} [Minecraft] (minecraft.jar) 
	UCHIJ	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJ	FML{8.0.99.99} [Forge Mod Loader] (forge-1.11.2-13.20.0.2228-universal.jar) 
	UCHIJ	forge{13.20.0.2228} [Minecraft Forge] (forge-1.11.2-13.20.0.2228-universal.jar) 
	UCHIJ	ftbl{0.0.0} [FTBLib] (FTBLib-1.1x-3.6.3.jar) 
	UCHIE	ftbu{0.0.0} [FTBUtilities] (FTBUtilities-1.1x-3.6.3 .jar) 
	Loaded coremods (and transformers): 
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.5.0 NVIDIA 367.57' Renderer: 'GeForce 920MX/PCIe/SSE2'`

**Comment by DarthJimBob** [comment]
Gist with full log here: https://gist.github.com/DarthJimBob/357f187881b07afa700c670959b27c94

**Comment by LatvianModder** [comment]
Install CompatLayer

**Comment by DarthJimBob** [comment]
Maybe that should be listed as a dependency somewhere.

**Comment by XvSkItZvX** [comment]
I would like to comment my agreement with the above. Five months after this issue presented was resolved and closed, I ran into this issue while make a 1.10.2 pack. And the only reason I was able to find this solution, which did work, was because I googled "Caused by: java.lang.NoClassDefFoundError: mcjty/lib/tools/ItemStackTools"...and this was the top link. Seems like it would be easier on your to prevent this by listing the dependency on the dependency page at least.
